.github/workflows/deploy_server.yaml

```diff
@@ -49,23 +50,25 @@ jobs:
 
     - name: "build driver image"
       run: |
-        docker build -f driver/Dockerfile.hail --build-arg HAIL_SHA=$HAIL_SHA --tag $DRIVER_IMAGE driver
+        docker build -f driver/Dockerfile.hail --build-arg HAIL_SHA=$HAIL_SHA --tag $DRIVER_IMAGE:$IMAGE_TAG driver
 
     - name: "push driver image"
       run: |
-        docker push $DRIVER_IMAGE
+        docker push $DRIVER_IMAGE:$IMAGE_TAG
+        docker tag $DRIVER_IMAGE:$IMAGE_TAG $DRIVER_IMAGE:latest
+        docker push $DRIVER_IMAGE:latest
 
     - name: "build server image"
       run: |
-        docker build --build-arg DRIVER_IMAGE=$DRIVER_IMAGE --tag $SERVER_IMAGE server
+        docker build --build-arg DRIVER_IMAGE=$DRIVER_IMAGE:$IMAGE_TAG --tag $SERVER_IMAGE:$IMAGE_TAG server
 
     - name: "push server image"
       run: |
-        docker push $SERVER_IMAGE
+        docker push $SERVER_IMAGE:$IMAGE_TAG
```

Comment:
Should we also have `latest` here for consistency?

Reply:
Done!